Service/Request/RequestFactory.php

```diff
@@ -123,18 +123,21 @@ private function getRequestUri(ServiceRequestInterface $serviceRequest)
         $endpoint = $this->endpointRegistry->getEndpoint($serviceRequest);
         $baseUrl = $endpoint->getBaseUrl();
         $path = $endpoint->getPath();
+        $queryParams = $this->parseQueryParams($path);
 
         //check for placeholders
         preg_match_all('/{(\w*)}/', $path, $matches);
         foreach ($matches[0] as $index => $placeholder) {
-            $getterMethod = 'get'.ucfirst($matches[1][$index]);
+            $key = $matches[1][$index];
```

Comment:
This change doesn't seem necessary.

Reply:
Also could we use Symfony property access here?

Reply:
@daydiff , I use this `$key` for checking is the match found in query parameters or not. (https://github.com/auto1-oss/service-api-client-bundle/pull/10/files/9ea0dee0e89dbd08ee7a30c24aeb1cfa30df9c21#diff-8a903834952ccfd3a4e15d94ba4d5f19R140)

Reply:
Guys, it's a BUG fix. Let's not ask for a new functionality/not related to the fix improvements.

Reply:
> Also could we use Symfony property access here?

It's a bug fix, let's not ask for more than fixing the bug properly.